utils/check-environment-variable.ts

```diff
@@ -0,0 +1,12 @@
+/*
+* Check if the environment variables are set
+* @param args - Environment variables to check
+* @throws Error if any of the environment variables is not set
+* @returns void
+* */
+export function checkEnvVariables(...args: string[]) {
+    const missingVariables = args.filter(arg => process.env[arg] === undefined);
+    if (missingVariables.length > 0) {
+        throw new Error(`The following environment variables are not set yet: ${missingVariables.join(', ')}. Please set them in .env file or pipeline environments.`);
```

Comment:
**Suggestion:** Consider using a more descriptive error message that includes the name of the missing environment variables.
> 🌴  **Medium Priority** 
 > [category: enhancement, importance: 5]
> **Reason:** Enhancing the error message improves clarity for debugging, but it's a minor change in terms of overall impact on the codebase.
 

```suggestion
        throw new Error(`Missing environment variables: ${missingVariables.join(', ')}. Please set them in the .env file or pipeline environments.`);
```